sdks/how-to/how-to-migrate-to-icure-premium-features.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: how-to-migrate-to-icure-premium-features
+description: How to Migrate to iCure Premium Features
+tags:
+- Open Source
+- Migration
+- Premium Features
+---
+
+# How to Migrate to iCure Premium Features
+
+:::note
+
+This how-to assumes that you have been using the iCure Medical Device SDK in your application with the open-source version
+ of the iCure backend and that, after creating an application, a database, and a suer using the iCure Cockpit, you now
+ want to use the cloud version of iCure.
+
+:::
+
+If you were using the open source version of the iCure backend, and you now are migrating to the cloud version, there are
```

Comment:
My comment is therefore a bit duplicated from what you say here, we need to rework a bit :-)